photon-client/src/main/scala/com/linkedin/photon/ml/estimators/GameParams.scala

```diff
@@ -481,6 +489,10 @@ object GameParams {
         .text("An optional directory to output statistics about the training data")
         .foreach(x => params.summarizationOutputDirOpt = Some(x))
 
+      opt[Boolean](CHECK_DATA)
+        .text(s"Whether to check the data extensively or not (time consuming), default: ${defaultParams.checkData}")
+        .foreach(x => params.checkData = x)
```

Comment:
Should this be part of a greater 'data validation' parameter (like in Photon)? In either case, I think the parameter text should be more descriptive (what constitutes extensively, why is it time consuming?)